canonical/ubuntu.com
# Added updates service

## Done

- Readded updates service to isolate security team updates from web traffic

## QA

- Deploy to staging and confirm that the https://ubuntu.com/security/updates/cves.json url returns 409 errors consistently
